src/app/content/highlights/hooks/loadMore.ts

```diff
@@ -56,7 +56,7 @@ export const hookBody: ActionHookBody<
       highlights = await loadMore(services, summaryPageSize);
     } catch (error) {
       services.dispatch(toggleSummaryHighlightsLoading(false));
-      throw ensureApplicationErrorType(error, new HighlightPopupLoadError({ destination: 'myHighlights' }));
+      throw ensureApplicationErrorType(error as Error, new HighlightPopupLoadError({ destination: 'myHighlights' }));
```

Comment:
you could change the `ensureApplicationErrorType` helper to accept `any` as its first argument and internally do an `instanceof Error` check, that'd be more correct and have less changes to all these places